### PR TITLE
ayatana-settings: Handle missing org.gnome.desktop.interface::color-scheme

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: ayatana-settings [hunk 1]
```diff
@@ -45,14 +45,26 @@ class AyatanaSettings:
...
+
+            if bFound:
+
+                self.pGnomeSettings = Gio.Settings.new ('org.gnome.desktop.interface')
+                self.pGnomeSettings.connect ('changed::color-scheme', self.onColorSchemeChanged)
+                self.onColorSchemeChanged (self.pGnomeSettings, 'color-scheme')
+
+            else:
+
+                logger.warning ('org.gnome.desktop.interface::color-scheme not found. The application will not react to native dark/light mode changes.')
```
Comment (sunweaver) [inline ayatana-settings:63]: I'd rather silently ignore color-scheme not being present. That is: hide the setting in the UI and don't have to bother with the setting not working.

**Review by sunweaver — COMMENTED** [review]
Maybe engineer this differently and silently ignore the missing color-scheme? This implies hiding everything that belongs to it in the UI of ayatana-settings...

File: ayatana-settings (hunk 1 above)
Comment (tari01) [reply ayatana-settings:63]: There is nothing to hide. This code automatically switches the application to dark/light mode whenever the gschema key changes. This is a must-have for all Gtk 3.24+ applications. The dconf-editor itself is a good example.

Comment (tari01) [reply ayatana-settings]: I have removed the warning, though.

**Review by sunweaver — APPROVED** [review]
LGTM

**Pull request merged**